### PR TITLE
Feat/68/Add_division_view

### PR DESCRIPTION
**Pull request by Zhykovskiy**

### CODE REVIEW

**Review by vladtymo — CHANGES_REQUESTED** [review]
Let's use the reactive form.

File: MilitaryManager/client-app/src/app/shared/services/core/http.service.ts [hunk 1]
```diff
@@ -30,25 +38,25 @@ export class HttpService {
 
     get(path: string, params: HttpParams = new HttpParams()): Observable<any> {
         return this.httpClient
-            .get(`${path}`,  { params })
+            .get(`${path}`,  { headers: this.setHeaders(), params })
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/shared/services/core/http.service.ts]: What was the reason to make these changes?

File: MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private _fb: FormBuilder) { }
+
+  ngOnInit(): void { 
+    this._divisionsService.GetAllDivisions().subscribe((divisions)=>{this.divisions = divisions});
+  }
+
+  addDivision() {
+    if(this.addDivisionForm.valid) {
+
+      this.useRedClass = false;
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts]: Where do you use this property?

File: MilitaryManager/client-app/src/app/shared/services/core/http.service.ts (hunk 1 above)
Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/shared/services/core/http.service.ts]: These are changes from branch 78. I'll wait for its merge before my merge.

File: MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts (hunk 2 above)
Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts]: Deleted it.

**Review by vladtymo — APPROVED** [review]

**Review by Alex-knu — APPROVED** [review]

**Pull request merged**